### ISSUE TITLE
Some warnings.

### ISSUE THREAD

**Issue by nicolafiorillo** [bryanjos/joken#147]
Not a big deal but compiling from clean build I get some warnings:
```
==> joken
Compiling 6 files (.ex)
warning: variable cause is unused
  lib/joken/signer.ex:180

warning: function :jsx.decode/2 is undefined (module :jsx is not available)
  lib/joken/signer.ex:189

Generated joken app
```

**Comment by bryanjos** [comment]
Thanks, would gladly accept a PR for the first warning. The second one is a bit tricky because we support the jsx module optionally. I know there is a way to only compile or use things if an elixir module is not loaded, but not sure if the applies to an erlang module (I just haven't tried).

**Comment by bryanjos** [comment]
Closed with #148
